README.md

```diff
@@ -9,13 +9,35 @@ There are also jobs configured for accessing the Snappy Product Integration envi
 
 ## Provision
 
-There are provision scripts for creating the CI environment locally and in the cloud.
+There are provision scripts for creating the CI environment locally and in the cloud. These scripts use docker-machine and docker-compose, so you need to have them installed, see [https://docs.docker.com/machine/install-machine/](here) and [https://docs.docker.com/compose/install/](here). Once the provision is finished you can use the common commands of machine and compose, for instance, to list the machines:
```

Comment:
docker-machine isn't yet in the archive, and the version of docker-compose is old for the format that we are using here.